pkg/apis/autoscaling/v1alpha1/pa_types.go

```diff
@@ -113,6 +113,12 @@ type PodAutoscalerStatus struct {
 	// state of the world.
 	// +optional
 	Conditions duckv1alpha1.Conditions `json:"conditions,omitempty"`
+
+	// ObservedGeneration is the 'Generation' of the Configuration that
```

Comment:
Oops, copy/pasted the type names 🤦‍♂️

Reply:
yeah, I wasn't sure about those either, but I though you know better :)